### DISCUSSION TITLE
How to make "Capture to Tags" look in the front matter ?

### DISCUSSION THREAD

**Discussion by lyndhurst** [chhoumann/quickadd#456] [General]
Hi,

I saw from the changelog that releases 0.19.1 and 0.19.2 added the ability to look for tags in the front matter for the "Capture To Tag" feature.

I would like to use QuickAdd to insert links to newly created notes into a structure note, often called MOCs in the Obsidian community. I tag my structure notes, with the `index` keyword in an array field located in the YAML frontmatter under the `tags` key. I do not prefix my tags with a `#` sign in this array; and I know that some people use the YAML key `keyword` instead of `tags`.

Anyway, when I add `#index` in the `Capture To filename` field, an error is thrown stating that no notes tagged `index` were found. When I add `index` instead, of course it complains that the file does not exist, or creates it depending on the settings.

I tested adding a `#index` tag to a couple of notes, and they end up being listed in the suggester which indicates the `Capture To Tag` functionality works fine.

I also tested adding `#index` in the frontmatter `tags` array wondering if the plugin was looking for some kind of regex exact match, but it still did not work, and also messed up the YAML highlighting, teaching me that this is the YAML comment syntax :)

Finally my questions are:

- Is there a specific syntax to respect when adding the tag in the `Capture to` field for the plugin to look into the front matter ?
- Or, is the plugin looking for another YAML key than `tags` ?
- Or finally, am I missing something here ?

Thanks for your help.

**Comment by chhoumann** [comment]
Hi @lyndhurst!

This seems to be a bug. I had underestimated the permutations of YAML frontmatter.
Issue to track this: #459.

It's not you that's missing anything. Just a bug.

**Reply by chhoumann** [reply]
Just pushed a fix. Should be available in the next update :)

**Comment by lyndhurst** [comment]
Thanks for clarifying, and taking care of this issue so quickly.

Looking at #459, I see that you added a code block with an example of a YAML array, I used the other syntax so far in my notes:

```yaml
tags: [ tag1, tag2 ]
```

Do you think it is going to be an issue on my end to use the `Capture To Tags` feature ?

**Reply by chhoumann** [reply]
Should work just fine, based on my tests.

**Reply by lyndhurst** [reply]
Can't wait for the next update then, thanks again :+1:

**Reply by chhoumann** [reply]
Should already be out. Sure thing!

**Comment by lyndhurst** [comment]
Working great now, thank you !
